bofire/strategies/samplers/universal_constraint.py

```diff
@@ -37,9 +37,7 @@ def _ask(self, candidate_count: int) -> pd.DataFrame:
             fixed_experiments=self.candidates,
         )
 
-        samples = samples.iloc[
-            self.num_candidates :,
-        ]
+        samples = samples.iloc[self.num_candidates :,]
```

Comment:
I think this is a problem for the formatter, just revert it to the original one and then it should be fine.

Reply:
Sure, I just committed it!